sources/lib/util/options.ml

```diff
@@ -132,12 +132,15 @@ let vno_backward = ref false
 let vno_sat_learning = ref false
 let vinstantiate_after_backjump = ref false
 let vdisable_weaks = ref false
-let vinput_format = ref "why"
 let vanswers_with_loc = ref true
 
-type output = ONative | OSmtlib | OSZS
+type input = INative | ISmtlib | IWhy3 (* | ISZS *)
+let vinfer_input_format = ref true
```

Comment:
I think it would be nice if we started documenting not only the interface of modules (for which I prasie this PR which adds the necessary doc comments, :) ), but also the inside of modules, for instance, by adding a comment stating what those reference are supposed to hold (semantics, goals, etc...). This would really immensely help future code modifications (by instance to spot bugs if the use of the ref doe snot correspond to its semantics/intended goal, etc..).